dxr/templates/nunjucks/promoted.html

```diff
@@ -0,0 +1,27 @@
+{% macro promoted_list(query, promoted, promoted_count, www_root, tree) -%}
+  {% for result in promoted %}
+    <div class="result-head">
+      <div class="left-column">
+        <div class="icon-container {{ result.iconClass }}"></div>
+      </div>
+      <div>
+        {{ result.pathLine | safe }}
```

Comment:
No, that's okay. Your reasoning shows that it should really be safe. (A comment here laying out that same reasoning would not go amiss.)